hr_employee_gravatar/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================================
+Employees Synchronize Gravatar image
+====================================
+
+Extends Synchronize Gravatar image to employees
+
+Installation
+============
+
+To install this module, you need to:
+
+* base_user_gravatar
+
+* hr
+
+Usage
+=====
+
+To configure this module, you need to:
+
+* go to user profile
```

Comment:
I would add something like press **Edit button** or I would replace **go to user profile** by **edit user profile**.